.env

```diff
@@ -0,0 +1 @@
+CLOUDINARY_URL=cloudinary://684891284735241:VIvo708rJV0nA20uuBW_RA3gRzA@dlvvjdcud
```

Comment:
are we supposed to see this Key on GitHub? I think you sould have put git ignore?